### DISCUSSION TITLE
On macOS 14, it is now natively supported to disable mouse acceleration.

### DISCUSSION THREAD

**Discussion by micsama** [linearmouse/linearmouse#516] [General]
<img width="751" alt="image" src="https://github.com/linearmouse/linearmouse/assets/41811005/4a0a3dc9-58c4-4d00-9e63-8ef5117bb733">

**Comment by lujjjh** [comment]
Yeah, [LinearMouse v0.9.2](https://github.com/linearmouse/linearmouse/releases/tag/v0.9.2) has started to use the new API on macOS Sonoma. And you can still take advantage of device-specific and app-specific settings in LinearMouse.

Related discussion: #493.
